examples/python/observability/helloworld_pb2.pyi

```diff
@@ -0,0 +1,17 @@
+from google.protobuf import descriptor as _descriptor
```

Comment:
Please add all generated files to the [Makefile](https://github.com/grpc/grpc/blob/master/examples/python/Makefile).